spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/DataSet.scala

```diff
@@ -47,7 +50,7 @@ object Cifar10DataSet extends ResNetDataSet {
     DataSet.array(Utils.loadTrain(path))
       .transform(BytesToBGRImg())
       .transform(BGRImgNormalizer(trainMean, trainStd))
-      .transform(HFlip(0.5))
+      .transform(JHFlip(0.5))
```

Comment:
why we touch cifar10?

Reply:
Because the CFAIR still applies the older transformers which has the same naming as the opencvs ones.  So I rename the legacy ones in import statements. 

Check 
```scala
import com.intel.analytics.bigdl.dataset.image.{HFlip => JHFlip}
```